code/style.css

```diff
@@ -43,20 +49,74 @@ select {
 .guess-who-icon {
   width: 126px;
 }
+/* Aside section: player, timer, questions tracker  */
+
+.question-tracker {
+  color: white;
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+.timer-wrapper {
+  margin-top: 5px;
+  color: white;
+
+}
+.name-wrapper {
+  margin-top: 15px;
+  margin-bottom: 0;
+  color: white;
+}
+/* Board styling */
```

Comment:
move down to line 70 adding a space between comments and previous grouped elements, like you've done before